guide/14-deep-learning/how_estimate_batch_size_works.ipynb

```diff
@@ -0,0 +1,251 @@
+{
```

Comment:
<p>The <code>estimate_batch_size</code> class accepts the following paramters: </p><ul><li><code>model</code>&nbsp;(Required Parameter): Required&nbsp;<code>arcgis.learn</code>&nbsp;imagery model. The model instance for which batch size should be estimated. Not supported for text, tabular, timeseries, or tracking models like <code>FullyConnectedNetwork</code>, <code>MLModel</code>, <code>TimeSeriesModel</code>, <code>SiamMask</code>, <code>PSETAE</code> and <code>EfficientDet</code> models.</li><li><code>mode</code>&nbsp;(Optional Parameter): Optional string. Default train. The mode for which batch size is estimated. Supports&nbsp;<code>train</code>&nbsp;and&nbsp;<code>eval</code>&nbsp;modes for calculating batch size in training mode and evaluation mode respectively. Note: <code>max_batchsize</code> is capped at <code>1024</code> for train and eval mode and <code>recommended_batchsize</code> is capped at <code>64</code> for train mode.</li><li><br></li><li class="ql-indent-1"><code>train</code> : To to be used when the user wants to estimate batch size for training a model.</li><li class="ql-indent-1"><code>eval</code> : To be used when the user also wants to estimate batch size for inferencing on a trained model.</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1533/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>